media-sound/ladish/ladish-1.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit python-single-r1 waf-utils
+
+DESCRIPTION="LADI Session Handler - a session management system for JACK applications"
+HOMEPAGE="https://ladish.org"
+
+SRC_URI="https://dl.ladish.org/ladish/${P}-g36c489e4.tar.bz2"
+S="${WORKDIR}/${P}-g36c489e4"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+IUSE="debug +lash"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+RDEPEND="
+	virtual/jack
+	media-sound/jack2[dbus]
+	sys-apps/dbus
+	dev-libs/expat
```

Comment:
```suggestion
	dev-libs/expat
	media-sound/jack2[dbus]
	sys-apps/dbus
	virtual/jack
```
Package Atom's should be sorted

Reply:
Sorting dependencies is currently(?) documented as optional. I'm ladish and jackdbus upstream and find virtual/jack and jackdbus as semantically more "core" dependencies than libdbus and expat, which are just used to implement the functionality required/provided.

In ladish-1.x (this pull request), the subgraphing/segmentation feature is implemented by the jmcore daemon, which links to libjack.so. Thus there is dependency on both vitrual/jack (for libjack.so dependency of jmcore) and jackdbus (media-sound/jack2[dbus]) as *the *core runtime dependency.

It makes some (when I put the upstream hat at least) sense to move virtual/jack at the end of the list, as the plan is to drop that for ladish-2. In future, ladish-2, WIP, (in setups with pipewire providing libjack.so and libjackserver.so) the plan is to use functionality for subgraphing/segmentation of (JACK app) nodes that is part of pipewire&jackdbus-3 combo.

Reply:
We'd really prefer them sorted for proxied maintainers at least. The only other acceptable order is to match build system checks.